@here/harp-materials/lib/MapMeshMaterials.ts

```diff
@@ -536,22 +536,27 @@ export namespace FadingFeature {
      * actual camera setup.
      * @param fadeNear The fadeNear value to set in the material.
      * @param fadeFar The fadeFar value to set in the material.
-     * @param updateUniforms If `true`, the fading uniforms are set. Not required if material is
-     *          handling the uniforms already, like in a [[THREE.ShaderMaterial]].
-     * @param additionalCallback If defined, this function will be called before the function will
-     *          return.
      */
     export function addRenderHelper(
         object: THREE.Object3D,
```

Comment:
This chould be changed to take a TileObject, to simplify access to the material.

Reply:
TIleObject is  in MapView so we would end with circular dependency ... or unrelated refactor.
Anyway, this code will be removed.